polymer.json

```diff
@@ -24,13 +22,16 @@
   },
   "builds": [
     {
-      "preset": "es5-bundled"
+      "preset": "es5-bundled",
+      "basePath": true
```

Comment:
Adding `"basePath": true` will make the build not working on a standard file server. See this comment: https://github.com/Polymer/prpl-server-node/issues/24#issuecomment-308537285

Reply:
Possible fix is to include the no `basePath` config as well, like https://github.com/Polymer/shop/blob/firebase/polymer.json, but you still need to know to use the `static-es*` builds instead of the `es*` builds.